examples/online-shop/update_service.bal

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/sql;
+
+service /onlineshop/update on onlineShopListener {
+    resource function post customers(@http:Payload Customer customer) returns string|error? {
```

Comment:
```suggestion
    resource function post customers/[int id](@http:Payload Customer customer) returns string|error? {
```

Reply:
removed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/36/commits/9c39326fb639c7553dd84fc179accd374e46f2ff